addons/binding/org.openhab.binding.hpprinter/src/main/java/org/openhab/binding/hpprinter/internal/handler/DefaultRequestHandler.java

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.hpprinter.internal.handler;
+
+import static org.openhab.binding.hpprinter.internal.HPPrinterBindingConstants.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.openhab.binding.hpprinter.internal.config.HPPrinterConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+
+/**
+ * Type 1 Printer
+ *
+ * @author Stewart Cossey - Initial contribution
+ */
+@NonNullByDefault
+public class DefaultRequestHandler extends PrinterInfo {
+
+    private final Logger logger = LoggerFactory.getLogger(DefaultRequestHandler.class);
+    private final String endpointStatus = "/DevMgmt/ProductStatusDyn.xml";
+    private final String endpointUsage = "/DevMgmt/ProductUsageDyn.xml";
+    private final String endpointRestart = "/ProductActions/PowerCycle";
+
+    DefaultRequestHandler(HPPrinterConfiguration config, Thing thing) {
+        super(config, thing);
+        logger.debug("Using SVN_IPG_LEDM_119 parser for {}", thing.getUID().toString());
```

Comment:
You don't have to call `toString` it will be called automatically, but only when the logging-level actually applies.